subresource-loading.md

```diff
@@ -333,9 +358,11 @@ There are a number of different possible valid designs for which chunks a browse
 
 **A**: In the above proposal, chunk IDs are the atomic unit of loading and caching. The browser either uses whole chunk or it does not. Chunk IDs are abstract units of loading, not necessarily corresponding to a library/package: there may be multiple packages in a chunk, or one package may be divided into multiple chunks. If you want to be able to keep around just part of a chunk ID when only part of it changes, divide it into two chunk IDs ahead of time. Bundlers are responsible for making this metadata volume vs caching tradeoff.
 
+<!-- Can there be multiple chunks in a scoped directory? Or are developers expected to break up the directory into subdirectories in such cases? -->
```

Comment:
Yes; there is no relationship between chunks and directory structure (just the overall path/scope limitation)

Reply:
In that case it might be interesting to show how the browser can know which URL in a certain scope maps to which chunk ID. I don't currently have a mental image of it'd work.

Reply:
Changed comment to my latest question